tests/ui/entry-point/issue-119209.rs

```diff
@@ -0,0 +1,4 @@
+type Foo<'a, 'b> = (i32, impl PartialEq); //~ ERROR `impl Trait` in type aliases is unstable
```

Comment:
can you rename the test to something descriptive like "has-bound-vars"? Mentioning the issue in a comment would be nice too. It would also be nice if you could clean up the test to remove as many errors as possible (for example, I doubt the undeclared lifetimes are necessary) while still triggering the ICE.